lib/api-client.js

```diff
@@ -23,7 +24,39 @@ module.exports.getAuthToken = function(config, agentName, socket) {
     };
     debug(url);
     debug(postdata);
-    return got.post(url, { body: postdata, json: true, headers: { 'user-agent': config.UA } })
+
+    var proxy = process.env.http_proxy;
+    var agent;
+    if (proxy) {
+        log('Using proxy server: ' + proxy);
+        if (url.startsWith('https') && proxy.startsWith('https')) {
+            agent = tunnel.httpsOverHttps({
+                proxy: {
+                    host: proxy
```

Comment:
The value passed the host property must only be the host part, without protocol and port.